api/analyticsd/pb/analytics.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+package api.analyticsd.pb;
+option go_package = "github.com/textileio/textile/v2/api/analyticsd/pb";
+
+import "google/protobuf/struct.proto";
+
+enum Event {
+  EVENT_UNSPECIFIED = 0;
+  EVENT_SIGN_IN = 1;
+  EVENT_ACCOUNT_DESTROYED = 2;
+  EVENT_KEY_ACCOUNT_CREATED = 3;
+  EVENT_KEY_USER_CREATED = 4;
+  EVENT_ORG_CREATED = 5;
+  EVENT_ORG_LEAVE = 6;
+  EVENT_ORG_DESTROYED = 7;
+  EVENT_ORG_INVITE_CREATED = 8;
+  EVENT_GRACE_PERIOD_START = 9;
+  EVENT_GRACE_PERIOD_END = 10;
+  EVENT_BILLING_SETUP = 11;
+  EVENT_BUCKET_CREATED = 12;
+  EVENT_BUCKET_ARCHIVE_CREATED = 13;
+  EVENT_MAILBOX_CREATED = 14;
+  EVENT_THREAD_DB_CREATED = 15;
+  EVENT_FIL_REWARD_RECORDED = 16;
+}
+
+enum AccountType {
+  ACCOUNT_TYPE_UNSPECIFIED = 0;
+  ACCOUNT_TYPE_DEV = 1;
+  ACCOUNT_TYPE_ORG = 2;
+  ACCOUNT_TYPE_USER = 3;
+}
+
+message IdentifyRequest {
+  string key = 1;
+  AccountType account_type = 2;
+  bool active = 3;
+  string email = 4;
+  google.protobuf.Struct properties = 5;
+}
+
+message IdentifyResponse {}
+
+message TrackRequest {
+  string key = 1;
+  AccountType account_type = 2;
+  bool active = 3;
+  Event event = 4;
+  google.protobuf.Struct properties = 5;
```

Comment:
Found this nice proto type `Struct` that models JSON data. In generated Go code, it ends up as `map[string]interface{}` which is nice and consistent with what the Segment API wants as well.